pages/src/_data/buildingsettings.yml

```diff
@@ -40,4 +40,29 @@ glassblower:
       - name: Priority
         description: This is the default setting. The hut will try to use the recipe that is higher up in their recipe list first.
       - name: Warehouse Stock
-        description: The hut will look in the warehouse first to see what resource you have more of before deciding what recipe it will use.
\ No newline at end of file
+        description: The hut will look in the warehouse first to see what resource you have more of before deciding what recipe it will use.
+guardtower:
+  - name: Task
+    description: This is where you can choose if you want the Guard to patrol, follow, or guard.
+    options:
+      - name: Patrol
+        description: The guard will patrol an area you designate in <strong>Patrol Settings</strong>. 
+      - name: Guard
+        description: You can set one area for the Guard to stay in. When you click "Set Target", you will be taken to the <strong>Selection Tools</strong> tab, where you can get the the Guard Scepter and designate the guard location.
+      - name: Follow
+        description: The guard will follow you around as your personal bodyguard, protecting you or fighting alongside you. They will even go outside the colony when following! You can designate how they follow you in <strong>Follow Settings</strong>. 
+      - name: Patrol Mine
+        description: The Guard will patrol the level of a <a href="../../source/buildings/mine">Mine</a> the <a href="../../source/workers/miner">Miner</a> is currently mining at. You must also select the guard in the Mine GUI in order for the guard to being patrolling.
+  - name: Retreat on low health
+    description: Here you can choose if the Guard will retreat when they have low health, if possible. You can choose from Off or On.
```

Comment:
I would suggest shortening this to:

>     description: You can choose if the Guard will retreat when they have low health.

I'm curious why "if possible" is included. Are there cases where the guard can't/won't retreat at low health, even if it's on?

Reply:
That's what was previously listed there.  I'm guessing it doeesn't guarantee the guard can escape if they can't outrun the mob?